docs/_documentations/mdt-eclipse-debugproject.md

```diff
@@ -25,6 +25,9 @@ Codewind for Eclipse supports debugging Microprofile/Java EE and Spring projects
 - Select **Import Project**.
 2. If you need to debug any initialization code, set breakpoints in this code now. You can also set breakpoints in your application code at this time.
 
+### Tips
+1. If you want to take advantage of Java hot code replace and make changes to your code while debugging it is recommended to disable automatic builds. Right-click on your project in the **Codewind Explorer** view and select **Disable Auto Build**. If you want to start a build while automatic builds are disabled, right-click on your project and select **Build**. Remember to re-enable automatic builds when you are finished debugging (right-click on your project and select **Enable Auto Build**).
```

Comment:
What if we made this tip an optional step 3 under "Prerequisites" instead of giving it its own header?

I also made some minor changes to the text:

```
3. [Optional] If you want to use Java hot code replace and change your code while you debug, disable automatic builds.
- To disable automatic builds, right-click your project in the **Codewind Explorer** view and select **Disable Auto Build**.
- If you want to start a build while automatic builds are disabled, right-click your project and select **Build**.
- Enable automatic builds again after you finish debugging. To enable automatic builds again, right-click your project and select **Enable Auto Build**.
```

Reply:
Thanks for the review. Updated the file as suggested.